### DISCUSSION TITLE
Is there an efficient way to represent ALL partitions of one table in ‘alter partition’ SQL

### DISCUSSION THREAD

**Discussion by oxygenkun** [ClickHouse/ClickHouse#51714] [Q&A] [answered]
In the ALTER PARTITION query, I must specify all PARTITION|PART partition_expr. which is tedious because I need to request all partitions from `system.parts`, if I want to manipulate the whole table. 

I want to find an easy expression to represent ALL partitions of one table and use that in `partition_expr` part.

**Comment by UnamedRus** [comment, marked as answer]
No, there is no such option.

You can just generate list of commands via SQL and apply them at once.
